layers/osx/README.org

```diff
@@ -32,11 +32,7 @@ and we encourage you to do so :)
 
 ** Layer
 
-To use this configuration layer, add it to your =~/.spacemacs=
-
-#+BEGIN_SRC emacs-lisp
-  (setq-default dotspacemacs-configuration-layers '(osx))
-#+END_SRC
+To use this configuration layer, add it to your =~/.spacemacs=. You will need to add `osx` to the existing list of `dotspacemacs-configuration-layers` in this file.
```

Comment:
Agreed, but until then are we comfortable with newbies continuing to get snagged? This hit me last night and was incredibly frustrating. Until the solution is present, I think we should update the README to assist newcomers as stop gap. We can update the README when #2213 is resolved.

That's my 0.02, I almost dropped Spacemacs altogether over this, it was really frustrating.